src/components/panel/panel.spec.js

```diff
@@ -2713,7 +2713,7 @@ describe('$mdPanel', function() {
 
           var panelRect = document.querySelector(PANEL_EL)
               .getBoundingClientRect();
-          expect(panelRect.right).toBeApproximately(myButtonRect.left);
+          expect(panelRect.right).toBeApproximately(myButtonRect.left, 6);
```

Comment:
So the default of `1` fails on Ubuntu, I am guessing that you used 6 because 2, 3, 4, and 5 also fail?
It would be nice to add a comment to explain the magic number `6` here and below.

Were you able to identify the cause of this? Is this a Firefox bug on Ubuntu?

Reply:
No, I was unable to identify why this offset on Firefox/Ubuntu

Reply:
I logged the left and top dimensions computed in panel.js:MdPanelPosition.prototype._calculatePanelPosition and are different for some tests on Firefox and Chrome. 

I'll investigate a bit more to see where the gap comes from.

Reply:
I think that the problem comes from a different font size in FF in Ubuntu.

In the tests, the Hello World div is too large for the place between the left size of viewport+margin and the button, so the div is right moved to be placed in the viewport. 

I moved the button to the right to make more place for the div.

![img_20180405_214044](https://user-images.githubusercontent.com/9973512/38388045-4f53b7cc-391a-11e8-9df0-d385e576b1da.jpg)